### DISCUSSION TITLE
How can I contact the owner of a github profile?

### DISCUSSION THREAD

**Discussion by cipopescu-dev** [community/community#40057] [New to GitHub]
Hello! 
Long story short: I have a certain username on every internet account I use (insta, facebook, snap, reddit, etc).
Somehow, someone managed to use the exact same username on github, on an account made in 2018.
Looking at the contributions table, seems like that person does not use the account at all (or maybe everything is private?)

How can I contact that person in order to negociate his profile name? Perhaps he won't mind to use another username 🤝

**Comment by radar-blip** [comment]
They may have an email under their profile picture (or not).
And yes, they could have made all activity private.

**Comment by AlefMemTav** [comment]
Comment something in some repository or make a pullrequest? Try it.

**Comment by jfullstackdev** [comment]
let me comment on this one, that's a possibility that the other user will give you that handle

but I think, you can think of more than those handles that will be equivalent to your liking, I have so many handles on my private list that I can use, that are equally the same level as this what I'm using now and they are still free
